setup.cfg

```diff
@@ -31,3 +31,7 @@ include_trailing_comma = True
 indent = '    '
 line_length = 120
 multi_line_output = 3
+
+[MESSAGES CONTROL]
```

Comment:
disable pylint in the backend line instead of disabling everything

Reply:
solved